src/electron/add_tap_device.bat

```diff
@@ -80,9 +87,11 @@ call :wait_for_device "%DEVICE_NAME%"
 :: already enabled:
 ::   This network connection does not exist.
 ::
-:: So, continue even if this command fails - and always include its output.
+:: So, continue even if this command fails, attempt to enable it through powershell -
+:: and always include its output.
```

Comment:
"include its output" in what?  Is the purpose of this powershell command to ensure that some line shows up in the logs when this script runs?

Reply:
Include the output => do not redirect stderr or stdout to `>nul`.

Reply:
I would appreciate an expanded comment here.  This comment doesn't explain why the powershell command would ever be necessary, since the only failure it mentions is the case where the adapter is already enabled.  Maybe the comment should say

> The powershell command is used to ensure the adapter is enabled if the netsh command fails and leaves the adapter in a disabled state.  No such failure case has yet been observed, but this command would correct such a failure and should not cause problems otherwise.